datastream-server/src/main/java/com/linkedin/datastream/server/assignment/StickyMulticastStrategyFactory.java

```diff
@@ -20,15 +20,14 @@
  */
 public class StickyMulticastStrategyFactory implements AssignmentStrategyFactory {
   public static final String CFG_IMBALANCE_THRESHOLD = "imbalanceThreshold";
+  public static final Integer DEFAULT_IMBALANCE_THRESHOLD = 1;
```

Comment:
can you change the type to "int" instead of "Integer" now that the constructor directly expects an int? there are performance issues with using Integer where not necessary https://stackoverflow.com/questions/4624933/comparing-performance-of-int-and-integer